docs/ref/django-admin.txt

```diff
@@ -1299,7 +1299,7 @@ don't.
 
     This is a known limitation. It arises from the need to serialize objects
     in order to exchange them between processes. See
-    :ref:`python:pickle-picklable` for details.
+    :ref:`python:pickle-pickleable` for details.
```

Comment:
"picklable" is the spelling that the Python docs use (and we can't change the link name cause that'll break it)